From 16cc1bea1eca50caebace21c0674beb3c9f72ae0 Mon Sep 17 00:00:00 2001
From: Kei Okada <kei.okada@gmail.com>
Date: Sun, 15 Mar 2015 15:00:12 +0900
Subject: [PATCH] CMakeLists.txt: display doxygen for ros-release

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 31245ff..4127dea 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -161,7 +161,7 @@ add_subdirectory(idl)
 add_subdirectory(lib)
 add_subdirectory(ec)
 add_subdirectory(rtc)
-option(ENABLE_DOXYGEN "Use Doxygen" ON)
+option(ENABLE_DOXYGEN "Use Doxygen" OFF)
 if(ENABLE_DOXYGEN)
   add_subdirectory(doc)
 endif()
-- 
1.9.1

