From 811ad6ede8046eb0d3b2530a154441c6464e0521 Mon Sep 17 00:00:00 2001
From: Kei Okada <kei.okada@gmail.com>
Date: Sun, 15 Mar 2015 15:00:53 +0900
Subject: [PATCH] CMakeLists.txt: display doxygen for ros-release

---
 CMakeLists.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e69a43e..0fd4e1c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -168,7 +168,7 @@ add_subdirectory(idl)
 add_subdirectory(lib)
 add_subdirectory(ec)
 add_subdirectory(rtc)
-option(ENABLE_DOXYGEN "Use Doxygen" ON)
+option(ENABLE_DOXYGEN "Use Doxygen" OFF)
 if(ENABLE_DOXYGEN)
   add_subdirectory(doc)
 endif()
-- 
1.7.9.5

