From 748c600a4788874c3ca3397d65594e0f7f8e431e Mon Sep 17 00:00:00 2001
From: Kei Okada <kei.okada@gmail.com>
Date: Fri, 13 Feb 2015 01:17:27 +0900
Subject: [PATCH 2/5] fix for old OpenHRP3 that uses OpenHRP3 as python module
 name

---
 python/hrpsys_config.py |    7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/python/hrpsys_config.py b/python/hrpsys_config.py
index d02a6b5..9f0a154 100755
--- a/python/hrpsys_config.py
+++ b/python/hrpsys_config.py
@@ -4,7 +4,12 @@ import os
 import rtm
 
 from rtm import *
-from OpenHRP import *
+import imp
+try:
+    imp.load_module('OpenHRP')
+    from OpenHRP import *
+except:
+    from OpenHRP3 import * # for old OpenHRP3 (3.1.7)
 from hrpsys import *  # load ModelLoader
 
 import socket
-- 
1.7.9.5

