From d8f88654966ee47621b08c6c6caa2faca2ffa86a Mon Sep 17 00:00:00 2001
From: Kei Okada <k-okada@jsk.t.u-tokyo.ac.jp>
Date: Fri, 1 May 2020 13:20:53 +0900
Subject: [PATCH 8/8] remove .catkin and others from install

---
 CMakeLists.txt | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6f125d1..90d2ac7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -258,10 +258,12 @@ execute_process(COMMAND cmake -E make_directory share/hrpsys WORKING_DIRECTORY \
 execute_process(COMMAND cmake -E create_symlink ../../../hrpsys/idl     share/hrpsys/idl WORKING_DIRECTORY \$ENV{DESTDIR}\${CMAKE_INSTALL_PREFIX}/share/hrpsys)
 execute_process(COMMAND cmake -E create_symlink ../../../hrpsys/samples share/hrpsys/samples WORKING_DIRECTORY \$ENV{DESTDIR}\${CMAKE_INSTALL_PREFIX}/share/hrpsys)
 ")
-  install(CODE "file(REMOVE \"\$ENV{DESTDIR}\${CMAKE_INSTALL_PREFIX}/.catkin\")") ## remove .catkin from install
-  install(CODE "file(REMOVE \"\$ENV{DESTDIR}\${CMAKE_INSTALL_PREFIX}/.rosinstall\")") ##
-  install(CODE "file(REMOVE \"\$ENV{DESTDIR}\${CMAKE_INSTALL_PREFIX}/_setup_util.py\")") ##
-  install(CODE "file(REMOVE \"\$ENV{DESTDIR}\${CMAKE_INSTALL_PREFIX}/env.sh\")") ##
+  ## remove .catkin and others from install
+  install(CODE "
+  file(GLOB RM_FILES \"\$ENV{DESTDIR}\${CMAKE_INSTALL_PREFIX}/*\")
+  foreach(RM_FILE \${RM_FILES})
+    file(REMOVE \${RM_FILE})
+  endforeach()")
 endif()
 if(need_catkin)
   message(WARNING "install tricky force write catkin marker file")
-- 
2.17.1

